pkg/mac/scripts/horizon-cli-uninstall.sh

```diff
@@ -31,3 +31,14 @@ rm -f $DESTDIR/share/man/*/man1/hzn.1
 echo "Removing $SRCDIR/horizon-cli ..."
 rm -rf $SRCDIR/horizon-cli
 
+# remove hzn_bash_autocomplete.sh that agent-install.sh put in .bashrc and .bash_profile
+if [ -f "${HOME}/.bash_profile" ]; then
+        if [[ "A`grep 'hzn_bash_autocomplete' ~/.bash_profile`" != "A"  ]];then
```

Comment:
Why do you have to check this with `grep`? The sed command will do nothing if the autocomplete script is not found. Same comment applies to `.bashrc` below.

Reply:
@bmpotter , You are right, It sounds to be a little complicated. I will remove this condition. Thanks